### PR TITLE
Cast command line arguments to strings

### PR DESCRIPTION
**Pull request by hoechenberger**
Cast command line arguments to strings

Avoid the bug we saw yesterday with @agramfort

### Before merging …

- [x] Changelog has been updated (`docs/source/changes.md`)

### CODE REVIEW

File: docs/source/changes.md [hunk 1]
```diff
@@ -92,3 +92,5 @@ authors:
   ({{ gh(336) }} by {{ authors.agramfort }}, {{ authors.hoechenberger }}, and {{ authors.crsegerie }})
 - `epochs_tmin` and `epochs_tmax` were named incorrectly in some test config files.
   ({{ gh(340) }} by {{ authors.crsegerie }})
+- Passing subject, session, task, and run names that consist only of numbers to `run.py` doesn't throw an exception anymore, even if the names weren't double-quotes.
```
Comment (hoechenberger) [inline docs/source/changes.md:95]:
```suggestion
- Passing subject, session, task, and run names that consist only of numbers to `run.py` doesn't throw an exception anymore, even if the values weren't double-quoted.
```

File: docs/source/changes.md [hunk 2]
```diff
@@ -92,3 +92,5 @@ authors:
...
+  ({{ gh(374) }} by {{ authors.hoechenberger }})
```
Comment (hoechenberger) [inline docs/source/changes.md:96]:
```suggestion
  ({{ gh(375) }} by {{ authors.hoechenberger }})
```

**Comment by agramfort** [comment]
Does it work if subject starts with 0?

**Comment by hoechenberger** [comment]
> Does it work if subject starts with 0?

I don't think so, and there's no obvious way to fix it… unless if maybe we ask users to specify the subject ID as `sub-xxx`, this would remove the  ambiguity… But then we'd still have the same issue for session, run, and potentially task :(

**Comment by agramfort** [comment]
then I would say that if you detect that fire converted run or subject to
an int you give
an informative message on how to format the parameter on the command line to
keep it a string.

my 2c

>

**Comment by hoechenberger** [comment]
@agramfort I was actually mistaken, if there are trailing zeros, it works out of the box! And with the changes here, we now also handle numbers that come without trainling zeros. So I think it's a good thing to have! Let's wait for CI to come back

**Review by agramfort — APPROVED** [review]

**Comment by agramfort** [comment]
thx @hoechenberger

**Pull request merged**